contracts/periodic-prize-pool/PeriodicPrizePool.sol

```diff
@@ -332,32 +333,41 @@ abstract contract PeriodicPrizePool is Timelock, BaseRelayRecipient, ReentrancyG
 
   function redeemTicketsInstantlyFrom(
     address from,
-    uint256 tickets
-  ) 
-    external nonReentrant returns (uint256) 
+    uint256 tickets,
+    uint256 prepaidExitFee
```

Comment:
Since it's called `sponsoredExitFee` in the event, it should be named the same here in the params.